File: SDKs/gambitcomm.local/mimic/21.00/go/pkg/models/operations/protocolnetflowgettrace.go
package operations

import (
	"openapi/pkg/models/shared"
)

type ProtocolNetflowGetTracePathParams struct {
	AgentNum int32 `pathParam:"style=simple,explode=false,name=agentNum"`
}

type ProtocolNetflowGetTraceRequest struct {
	PathParams ProtocolNetflowGetTracePathParams
}

type ProtocolNetflowGetTraceResponse struct {
	ConfigNETFLOW *shared.ConfigNetflow
	ContentType   string
	StatusCode    int64
}

File: SDKs/gambitcomm.local/mimic/21.00/python/src/sdk/models/operations/protocol_netflow_get_trace.py
import dataclasses
from typing import Optional
from ..shared import confignetflow as shared_confignetflow


@dataclasses.dataclass
class ProtocolNetflowGetTracePathParams:
    agent_num: int = dataclasses.field(metadata={'path_param': { 'field_name': 'agentNum', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class ProtocolNetflowGetTraceRequest:
    path_params: ProtocolNetflowGetTracePathParams = dataclasses.field()
    

@dataclasses.dataclass
class ProtocolNetflowGetTraceResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    config_netflow: Optional[shared_confignetflow.ConfigNetflow] = dataclasses.field(default=None)
    
